compiler/pipes/analyze-performance.cpp

```diff
@@ -0,0 +1,636 @@
+// Compiler for PHP (aka KPHP)
+// Copyright (c) 2020 LLC «V Kontakte»
+// Distributed under the GPL v3 License, see LICENSE.notice.txt
+#include "compiler/pipes/analyze-performance.h"
+
+#include "common/algorithms/string-algorithms.h"
+#include "common/termformat/termformat.h"
+#include "compiler/compiler-core.h"
+#include "compiler/data/src-file.h"
+#include "compiler/inferring/public.h"
+#include "compiler/gentree.h"
+
+namespace {
+
+VertexPtr remove_conv_wrap(VertexPtr vertex) noexcept {
+  if (vk::any_of_equal(vertex->type(), op_conv_int, op_conv_float, op_conv_string, op_conv_array,
+                       op_conv_bool, op_conv_mixed, op_conv_drop_false, op_conv_drop_null)) {
```

Comment:
Maybe, to check for OpInfo::type(vertex->type()) == conv_op?
(also op_move is conv_op by the way, I don't know whether we should handle it here)